qatrack/faults/views.py

```
@@ -212,10 +212,10 @@ def get_fields(self, request=None):
         return fields
 
     def selected(self, obj):
-        return '<input type="checkbox" class="test-selected" data-fault="%s" title="%s"/>' % (
+        return mark_safe('<input type="checkbox" class="test-selected" data-fault="%s" title="%s"/>' % (
             obj.id,
             _("Check to include this fault bulk setting approval statuses"),
-        )
+        ))
```

Ruff (0.15.14)

[error] Use of `mark_safe` may expose cross-site scripting vulnerabilities

(S308)